thavalon-webapp/src/components/Register.tsx

```diff
@@ -49,15 +51,28 @@ export function Register(props: RegisterProps) {
         setModalIsOpen(false);
     }
 
+    /**
+     * Function for handling errors from the resolver. Currently if called, only error should
+     * have the key confirmPassword.
+     * @param data The errors caused by the resolver.
+     */
+    function onError(data: DeepMap<RegisterData, FieldError>) {
+        if (data.confirmPassword === undefined || data.confirmPassword.message === undefined) {
+            console.log("Form onError called with no errors.");
+            return;
+        }
+        setFormErrorMsg(data.confirmPassword.message);
+    }
+
     /**
      * On error, just prevent page reload - form handles showing errors.
      * @param data The data being sent on submit.
      * @param event The event caused by submission.
      */
     async function onSubmit(data: RegisterData) {
         // disable button on start of submit
-        // TODO: Also add loading image
-        setDisabled(true);
+        console.log("onSubmit");

```

Comment:
nit: if we're logging this, should the log message be more specific, since we might have many onSubmits in the future?

Reply:
that was for debugging, will remove